src/super_gradients/training/utils/distributed_training_utils.py

```diff
@@ -171,23 +171,32 @@ def wait_for_the_master(local_rank: int):
 
 
 def setup_gpu_mode(gpu_mode: MultiGPUMode = MultiGPUMode.OFF, num_gpus: int = None):
+    """[DEPRECATED in favor of setup_device] If required, launch ddp subprocesses.
```

Comment:
Add @deprecated decorator please instead of modifying the method content..

Reply:
I added it but dont ge the deprecated message. Do you think it's because super_gradients doesnt have a __version__ ?